internal/transport/http2_server.go

```diff
@@ -445,8 +447,8 @@ func (t *http2Server) operateHeaders(frame *http2.MetaHeadersFrame, handle func(
 		if logger.V(logLevel) {
 			logger.Errorf("transport: %v", errMsg)
 		}
-		t.controlBuf.put(&earlyAbortStream{
-			httpStatus:     400,
+		_ = t.controlBuf.put(&earlyAbortStream{
```

Comment:
My IDE warns about code that ignores errors, which is what this function returns. This is a common analysis in CI (via tools like [errcheck](https://github.com/kisielk/errcheck)), to prevent mistakes where code forgets to check errors. So I added the `_ = ` to make it explicit that I am ignoring the returned error (which suppresses the yellow indicator in GoLand).

I am happy to undo this if you want.

Reply:
We generally don't prefer the `_ = ` option. If and where it makes sense, we prefer adding a small comment saying why it is safe to ignore the error. So, if you could do that, that would be great.